### ISSUE TITLE
Accessibility Features - Extended

### ISSUE THREAD

**Issue by egasparini** [yalla-coop/cost-of-living-support#51]
**Summary**
Accessibility is very important especially for our target audience, therefore we want to make sure that our websites are easily readable and accessible to anyone. 

Accessibility Features are available from the same top bar as the language switcher. 
The topic of accessibility is split in multiple tasks: yalla-coop/cost-of-living-support#52 about increasing the font size, and this task that covers more advanced features.

_Note: Accessibility features should be implemented both on UC & CoL._ 

**Wireframes**
Desktop: https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=5836%3A35532
Mobile: https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=4645%3A17521

**Acceptance Criteria
Front End**
- [x] Clicking on the Accessibility icons opens a separate page about accessibility. 
- [x] On the left side of the top bar nav there's a back arrow to go back to the previous page.  
- [x] Accessibility page has
   - [x] a description of what it is and the standard used
   - [x] a link to"contact us" in case of issues -> open their mail client to the email address [CoLdigital@hyde-housing.co.uk](mailto:CoLdigital@hyde-housing.co.uk) ~~or to the customised link if it's another company~~
   - [x] "Add colour overlay" section, with 4 colours: blue, green, yellow, red. Selecting one of the colours will add an overlay of that colour to the website. The colour changes as they select it, it doesn't close the page. 
   - [x] "Adjust Text Size" section, with a button to Increase/Decrease text size and explanation on how to do this on different browsers. Clicking on the button increases/decreases the text size across the website by 25%/back to normal.  
   - [x] Add ability to change font > users have a dropdown where they can choose between the default font Heebo (selected by default), Roboto and Comic Neue (had to change Arial to Roboto and Comic Neue as Comic Sans doesn't exist in google fonts @egasparini  
   - [x] "Text to Speech" section, with guides to this functionality on different browsers/devices

**UI Criteria**
**2 states:**
- [x] If there is no colour set, the background is white
- [x] If there is a colour set, the background is the selected one. This selected state stays on

Additional:
- [x] When you press the colour overlay section the background colour changes to the colour 
- [x] You can deselect the selected button to remove the coloured overlay
- [x] When you add overlay all text becomes Neutral / Main. Examples can be found here:
- https://www.figma.com/file/EqDZxsZf5o6cFfDbqyBsnt/Wireframes?node-id=5339%3A24928
- [x] All the links can be found in the Chiltern's accessibly page 

**Back End**
- not needed, just store font size / overlay colour in the local storage

**Comment by fadeomar** [comment]
agreed with @jema28 
  - to use Comic Neue instead of Comic Sans
  - to use Roboto instead of Arial

**Comment by egasparini** [comment]
As discussed with @fadeomar - as per the AC, the "contact us" email link should be customisable but we don't have a place anywhere for organisations to customise it. We should add this to the "Customise resources and phone numbers" page.

**Comment by jema28** [comment]
Just to check this @egasparini does this need a wireframe change?

**Comment by egasparini** [comment]
@jema28 if you have the time - go for it. If you don't it's okay. It should be a fairly straightforward change.

**Comment by jema28** [comment]
Oh hang on I think we decided this didn't have to be in there? Was this discussed with Maggie since @egasparini?

**Comment by egasparini** [comment]
@jema28 nope, we didn't discuss it with Maggie. Good catch! 

@maggiehoughton could you please confirm if the email address associate to the "contact us" link on the Accessibility page should be customisable by organisations? 
![Screenshot 2022-11-30 at 12 13 40](https://user-images.githubusercontent.com/55511970/204782115-34c90e70-f8e6-4434-ad1a-fc073677f34a.png)

Thank you!

**Comment by maggiehoughton** [comment]
Hm- I am not sure! I guess if there is an issue with the features of the app we would have to know so it should just be coldigital@hyde-housing.co.uk/ ucdigital@yde-housing.co.uk? If it's customised I guess a lot of people will just be directed to a CS type email in the host org who will not be able to help. So probably should not be able to customise?

**Comment by egasparini** [comment]
Yes - that makes sense. Thanks @maggiehoughton.
@fadeomar we don't need this task then. I'll close it and adjust the AC in the previous task for clarity.

**Comment by egasparini** [comment]
Sorry - I accidentally closed this task, I thought it was the follow up task. I've re-opened it 👍
